docs/sections/user_guide/cli/drivers/upp.rst

```diff
@@ -0,0 +1,106 @@
+``upp``
+=======
+
+The ``uw`` mode for configuring and running the `UPP <https://epic.noaa.gov/unified-post-processor/>`_ component.
+
+.. code-block:: text
+
+   $ uw upp --help
+   usage: uw upp [-h] [--version] TASK ...
+
+   Execute upp tasks
+
+   Optional arguments:
+     -h, --help
+         Show help and exit
+     --version
+         Show version info and exit
+
+   Positional arguments:
+     TASK
+       files_copied
+         Files copied for run
+       files_linked
+         Files linked for run
+       namelist_file
+         The namelist file
+       provisioned_run_directory
+         Run directory provisioned with all required content
+       run
+         A run
+       runscript
+         The runscript
+       validate
+         Validate the UW driver config
+
+All tasks take the same arguments. For example:
+
+.. code-block:: text
+
+   $ uw upp run --help
+   usage: uw upp run --cycle CYCLE --leadtime LEADTIME [-h] [--version] [--config-file PATH]
+                     [--batch] [--dry-run] [--graph-file PATH] [--quiet] [--verbose]
+
+   A run
+
+   Required arguments:
+     --cycle CYCLE
+         The cycle in ISO8601 format
```

Comment:
I would definitely second this suggestion, when first making a driver with a cycle I had to look up the actual format and failed once or twice before I found an example somewhere in our code.

Reply:
Good idea, will update. I'm considering the following text:
```
The cycle in ISO8601 format (e.g. 2024-05-08T18)
```
My reasoning:
1. The difference between upper- and lower-case letters in the proposed text above leaks internal Python details that I don't think belong in the CLI.
2. The underscore in the proposed text appears to work, but as far as I can tell is not ISO8601-compliant, as times are to be preceded by `T`. This may be a convenience added by Python.
3. If/when we need to support sub-hourly cycles, we may need to think about other assumptions in the codebase, so we may not want to suggest that we're correctly handling that level of precision today. Specifying a datetime precise to the hour is correct, ISO8601-compliant, and concise, so I've left off the `:MM:SS` part, as it would generally just be `:00:00` anyway.
4. I think we should continue to mention ISO8601 so that users understand that they have more power here should they choose to look it up. Also, it's a service to the NWP community to raise awareness of this international standard and encourage best practice (i.e. using it). IMO, anyone working with time-based data should be aware of ISO8601.

Reply:
Yep. I think that makes sense. I definitely just botched my suggestion, and this one looks good to me!

Reply:
Added ISO8601 example to CLI help.